build/signalr-scenarios.yml

```diff
@@ -64,6 +64,7 @@ steps:
       waitForCompletion: true
       messageBody: |
         {
-          "name": "benchmarksdriver",
-          "args": [ "${{ s.arguments }} --server ${{ parameters.server }} --client ${{ parameters.client }} --runtimeversion 5.0.* --self-contained --warmup ${{ parameters.warmup }} --duration ${{ parameters.duration }} --collect-counters --description Trend/Latest --session $(session) --table SignalR --sql SQL_CONNECTION_STRING --aspnetcoreversion 5.0.0-preview.5.20255.6" ]
+          "name": "benchmarksdriver2",
+          "args": [ "${{ s.arguments }} --application.endpoints ${{ parameters.server }} --load.endpoints ${{ parameters.client }} --runtimeversion 5.0.* --self-contained --warmup ${{ parameters.warmup }} --duration ${{ parameters.duration }} --collect-counters --description Trend/Latest --session $(session) --table SignalR --aspnetcoreversion 5.0.0-preview.5.20255.6 --config https://raw.githubusercontent.com/aspnet/Benchmarks/master/src/BenchmarksDriver2/benchmarks.signalr.yml" ]
```

Comment:
We need to use a new table. We can also get rid of the aspnetversion, I changed it on the server in the meantime.